JSONCoding/Classes/JSONUnarchiver.swift

```diff
@@ -27,7 +27,7 @@
 public protocol JSONUnarchiving {
     func unarchived<T>(with json: Any) throws -> T where T: JSONCoding
     func unarchived<T>(map jsons: [Any]) throws -> [T] where T: JSONCoding
-    func unarchived<T>(discardingErrorsMap jsons: [Any]) throws -> [T] where T: JSONCoding
+    func unarchived<T>(discardingErrorsMap jsons: [Any]) -> [T] where T: JSONCoding
```

Comment:
The actual implementation was not calling any `throw`ing functions, so I've removed this `throws` here to match.